mantle/cmd/kola/options.go

```diff
@@ -124,6 +124,7 @@ func init() {
 	bv(&kola.GCPOptions.ServiceAuth, "gcp-service-auth", false, "for non-interactive auth when running within GCP")
 	sv(&kola.GCPOptions.JSONKeyFile, "gcp-json-key", "", "use a service account's JSON key for authentication (default \"~/"+auth.GCPConfigPath+"\")")
 	bv(&kola.GCPOptions.Confidential, "gcp-confidential-vm", false, "create confidential instances")
+	sv(&kola.GCPOptions.LocalSSD, "gcp-localssd", "", "attach local ssd disk, should be NVME or SCSI")
```

Comment:
Should we "validate" the values here before launching the instance or is this done by the GCP library?